### ISSUE TITLE
Add 'checked' :visited style to check boxes

### ISSUE THREAD

**Issue by cweber** [sunlightlabs/wwc#15]
For funsies!

**Comment by jalbertbowden** [comment]
lol. word. see any you think would fit? i figure a plain one, that i can just color fill....
i'm assuming you mean after they click from the nav to the section....perhaps watching for them to scroll away....?

**Comment by SLarrick** [comment]
@jalbertbowden maybe i'm misunderstanding your comment, but it would be after they click the checkbox, not after they click from the nav to the section.

**Comment by cweber** [comment]
Yeah. What @SLarrick said, basically giving people the feeling of being able to check the boxes. We were trying to be clever about using only CSS. Though a person wouldn't ever be able to uncheck a box, which could be annoying. I'm gonna leave this hanging for now. It's low priority.

**Comment by jalbertbowden** [comment]
click the checkbox.........do you mean this? 
<img width="273" alt="screen shot 2016-06-23 at 3 50 55 pm" src="https://cloud.githubusercontent.com/assets/46432/16317694/51e0b718-395a-11e6-8659-151c6b08be97.png">
as in, the user clicks that red box?
if not, what do you mean?

**Comment by SLarrick** [comment]
@jalbertbowden: Yes, that's what we mean.

**Comment by jalbertbowden** [comment]
ok...that box is implemented in a way that doesn't provide that functionality. maybe i can do something in javascript, but if not, have to do a lot of things to implement this...
